File: example1/src/main/java/com/example/grpc/Delays.java
package com.example.grpc;

import java.util.concurrent.TimeUnit;

public final class Delays {

    public static void sleep(int seconds) {
        try {
            TimeUnit.SECONDS.sleep(seconds);
        } catch (InterruptedException e) {
            throw new RuntimeException(e);
        }
    }
}

File: _kotlin/src/main/java/com/example/grpc/Delays.kt
package com.example.grpc;

import java.util.concurrent.TimeUnit;

public final class /*TODO*/ Delays {

    public static void sleep(int seconds) {
        try {
            TimeUnit.SECONDS.sleep(seconds);
        } catch (InterruptedException e) {
            Thread.currentThread().interrupt();
            throw new RuntimeException(e);
        }
    }
}
